WalletWasabi.Fluent/Models/UiTriggers.cs

```diff
@@ -48,4 +51,13 @@ public UiTriggers(WalletViewModel walletViewModel)
 	/// Triggers on subscription and when a transaction is processed to the wallet or the anon score target changed.
 	/// </summary>
 	public IObservable<Unit> PrivacyProgressUpdateTrigger => TransactionsUpdateTrigger.Merge(AnonScoreTargetChanged).Skip(1);
+
+	/// <summary>
+	/// Triggers when any of the coins starts/stops participating in a coinjoin
+	/// </summary>
+	public IObservable<Unit> WalletCoinsCoinjoinTrigger => Observable.Interval(TimeSpan.FromSeconds(1), RxApp.MainThreadScheduler)
+		.Select(_ => _wallet.Coins.Where(x => x.CoinJoinInProgress).ToList())
+		.Buffer(2, 1)
```

Comment:
Buffer? How is that working?

Reply:
José helped me with this. https://github.com/zkSNACKs/WalletWasabi/pull/9827

> Adds a trigger when any of the coins in a given wallet starts/stops participating in a coinjoin.
Can be used to calculate the available balance correctly.

Reply:
Every second it puts all the CJing wallet coins into the buffer, and store always the two most recent lists. It compares the two list if there is any change then it fires a trigger for `WalletCoinsCoinjoinTrigger`.

Reply:
ACK